stimela/recipe.py

```diff
@@ -442,10 +434,7 @@ def singularity_job(self, image, config, singularity_image_dir,
         od = cab.IODEST["output"]
 
         self.log_dir = os.path.abspath(self.log_dir or output)
-        log_dir_name = os.path.basename(self.log_dir or output)
-        logfile_name = '.currentjob.log'.format(name.split('-')[0])
```

Comment:
Again this breaks the logging system @SpheMakh. The container pipes its log into a separate file, which the xrun then reads. It cannot be self.logfile because then you have both the container and the xrun with open handles onto the same file and creates that vicious feedback loop that we had before. The logger should create the final log user logfiles.